pubspec.yaml

```diff
@@ -13,7 +13,9 @@ authors:
 - Victor Berchet <victor@suumit.com>
 description: Compute hashCode correctly for your objects
 homepage: https://github.com/QuiverDart/quiver_hashcode
+
 environment:
-  sdk: '>=1.8.0 <2.0.0'
+  sdk: '>=1.8.0 <3.0.0'
```

Comment:
Let's match what `package:quiver` does: https://github.com/google/quiver-dart/blob/master/pubspec.yaml

Specifically, Dart 2 support comes in version 2.0.0 and it also simultaneously drop Dart 1.0.0 support.

Reply:
OK, should I also bump the package version to `2.0.0` than?

Reply:
Sorry for being slow here (traveling). Just pushed an update. PTAL.